### PR TITLE
PIP 8 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/m/mister_gutsy.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Mister Gutsy
+ManaCost:2
+Types:Artifact Creature Robot Soldier
+PT:1/1
+T:Mode$ SpellCast | ValidCard$ Aura,Equipment,Vehicle | ValidActivatingPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast an Aura or Equipment spell, put a +1/+1 counter on NICKNAME.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/m/mister_gutsy.txt:5]: wrong valid

File: forge-gui/res/cardsfolder/m/mutational_advantage.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Mutational Advantage
+ManaCost:1 G U
+Types:Instant
+A:SP$ PumpAll | ValidCards$ Permanent.YouCtrl+HasCounters | RememberPumped$ True | KW$ Hexproof & Indestructible | SubAbility$ DBPrevent | SpellDescription$ Permanents you control with counters on them gain hexproof and indestructible until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/m/mutational_advantage.txt:4]: no point in using remembered for this

File: forge-gui/res/cardsfolder/m/mister_gutsy.txt (hunk 1 above)
Comment (tool4ever) [inline forge-gui/res/cardsfolder/m/mister_gutsy.txt:5]: CARDNAME

File: forge-gui/res/cardsfolder/h/harold_and_bob_first_numens.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Name:Harold and Bob, First Numens
+ManaCost:2 G
+Types:Legendary Creature Treefolk Mutant
+PT:3/3
+K:Vigilance
+K:Reach
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self+Creature | Execute$ DBReturn | TriggerDescription$ When CARDNAME dies, if it was a creature, return it to the battlefield. It's an Aura enchantment with enchant Forest you control and "Enchanted Forest has '{T}: Add three mana of any one color. You get two rad counters.'" NICKNAME loses all other abilities.
+SVar:DBReturn:DB$ ChangeZone | Defined$ TriggeredNewCardLKICopy | Origin$ Graveyard | Destination$ Battlefield | GainControl$ True | AnimateSubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | Types$ Enchantment,Aura | RemoveCardTypes$ True | RemoveAllAbilities$ True | Keywords$ Enchant Forest you control | Abilities$ SPAttach | staticAbilities$ STAura | Duration$ Permanent
+SVar:STAura:Mode$ Continuous | Affected$ Land.EnchantedBy | AddAbility$ ABMana
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/h/harold_and_bob_first_numens.txt:10]: Missing Description

`Description$ Enchanted Forest has '{T}: Add three mana of any one color. You get two rad counters.'`

File: forge-gui/res/cardsfolder/s/synth_eradicator.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
+Name:Synth Eradicator
+ManaCost:2 R
+Types:Artifact Creature Synth Soldier
+PT:3/3
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME attacks, exile the top card of your library. You may get {E}{E} (two energy counters). If you don't, you may play that card this turn.
+SVar:TrigExile:DB$ Dig | Defined$ You | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBChoice
+SVar:DBChoice:DB$ GenericChoice | Choices$ DBEnergy,DBPlay
+SVar:DBEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 2 | SubAbility$ DBCleanup | SpellDescription$ You get 2 energy counters
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/s/synth_eradicator.txt:9]: RememberPut$ + Condition after should be better

Comment (Simisays) [reply forge-gui/res/cardsfolder/s/synth_eradicator.txt]: Hmm this isn't working now

Comment (tool4ever) [reply forge-gui/res/cardsfolder/s/synth_eradicator.txt]: somehow I don't think you will Remember a "Card" :P

Comment (Simisays) [reply forge-gui/res/cardsfolder/s/synth_eradicator.txt]: Hmm this new version isn't working as well. Revert to previous version ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/s/synth_eradicator.txt]: I'll debug

Comment (tool4ever) [reply forge-gui/res/cardsfolder/s/synth_eradicator.txt]: your `Remembered$Amount` is only for cards...
imo just replacing "Card" with "Player" should have worked 💩

Comment (Simisays) [reply forge-gui/res/cardsfolder/s/synth_eradicator.txt]: thanks working now

File: forge-gui/res/cardsfolder/s/synth_eradicator.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Name:Synth Eradicator
+ManaCost:2 R
+Types:Artifact Creature Synth Soldier
+PT:3/3
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME attacks, exile the top card of your library. You may get {E}{E} (two energy counters). If you don't, you may play that card this turn.
+SVar:TrigExile:DB$ Dig | Defined$ You | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | Imprint$ True | SubAbility$ DBEnergy
+SVar:DBEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 2 | RememberPut$ True | Optional$ True | SubAbility$ DBPlay | SpellDescription$ You get 2 energy counters
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/s/synth_eradicator.txt:8]: remove old SpellDescriptions

**Review by tool4ever — APPROVED** [review]

**Pull request merged**